#### Code
```kotlin
package com.glassthought.sandbox

import gt.sandbox.util.output.Out
import kotlinx.coroutines.delay
import kotlinx.coroutines.runBlocking
import java.util.concurrent.CompletableFuture

val out = Out.standard()

fun main() = runBlocking {
  val future = CompletableFuture.supplyAsync {
    throw RuntimeException("original-exc-msg-from-supplyAsync-future-block")
    out.println("supplyAsync")
    "Jon Snow"
  }

  future
    .exceptionally { ex: Throwable? -> fallbackMaker(ex) }
    .thenAccept { x: String -> out.println(x) }

  delay(100)
  println()
}

private fun fallbackMaker(ex: Throwable?): String {
  return "Fallback Result (${ex?.message})"
}
```


echo "" >> ""
echo -n '```' >> ""
echo "kotlin" >> ""
cat "" >> ""
echo '```' >> ""
echo "" >> ""

#### Command to reproduce:
```bash
gt.sandbox.checkout.commit ${first_commit_hash_short:?} \
&& cd "${GT_SANDBOX_PREFIXED_PWD:?}" \
&& cmd.run.announce ""
```

#### Recorded output of command:
```txt
> Task :app:checkKotlinGradlePluginConfigurationErrors SKIPPED
> Task :app:compileKotlin UP-TO-DATE
> Task :app:compileJava NO-SOURCE
> Task :app:processResources NO-SOURCE
> Task :app:classes UP-TO-DATE

> Task :app:run
[2024-11-22T05:51:09.648821Z][elapsed-since-start:   47ms][tname:main/tid:1] Fallback Result (java.lang.RuntimeException: original-exc-msg-from-supplyAsync-future-block)


BUILD SUCCESSFUL in 813ms
2 actionable tasks: 1 executed, 1 up-to-date
```
